addons/mail/static/src/core/common/thread.js

```diff
@@ -120,13 +120,12 @@ export class Thread extends Component {
                     } else {
                         if (this.props.order === "desc") {
                             this.scrollableRef.el.scrollTop = 0;
-                            this.props.thread.scrollTop = 0;
                         } else {
                             this.scrollableRef.el.scrollTop =
                                 this.scrollableRef.el.scrollHeight -
                                 this.scrollableRef.el.clientHeight;
-                            this.props.thread.scrollTop = "bottom";
                         }
+                        this.props.thread.scrollTop = "bottom";
```

Comment:
It would be good to rename it to `scrollPosition` in master. Conceptually, `bottom` is not coupled to the scrollable element itself, but rather to the concept of the thread (where the new messages are displayed). The `scrollTop` property makes it difficult to understand this nuance, which makes the code harder to grasp.